homeassistant/components/flipr/__init__.py

```diff
@@ -28,6 +53,22 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     unload_ok = await hass.config_entries.async_unload_platforms(entry, PLATFORMS)
 
     if unload_ok:
-        hass.data[DOMAIN].pop(entry.entry_id)
+        hass.data[DOMAIN].pop(CONF_ENTRY_FLIPR_COORDINATORS)
 
     return unload_ok
+
+
+def detect_invalid_old_configuration(hass: HomeAssistant, entry: ConfigEntry):
+    """Detect invalid old configuration and raise error if found."""
+
+    def find_duplicate_entries(entries):
+        values = [e.data["email"] for e in entries]
+        _LOGGER.debug("Detecting duplicates in values : %s", values)
+        return any(count > 1 for count in Counter(values).values())
+
+    entries = hass.config_entries.async_entries(DOMAIN)
+
+    if find_duplicate_entries(entries):
```

Comment:
Let's also raise an issue at the issue registry to tell what happened and what the user should do

Reply:
Done